controllers/keystoneapi_controller.go

```diff
@@ -195,8 +196,22 @@ func (r *KeystoneAPIReconciler) SetupWithManager(mgr ctrl.Manager) error {
 func (r *KeystoneAPIReconciler) reconcileDelete(ctx context.Context, instance *keystonev1.KeystoneAPI, helper *helper.Helper) (ctrl.Result, error) {
 	r.Log.Info("Reconciling Service delete")
 
+	// remove db finalizer before the keystone one
+	//get db pointer, then in database use ctlr.ult to remove it's finalizer
+
+	//TODO1 figure out where to get the service Name and Namepace for r.
+	// maybe use the func - GetKeystoneServiceWithName
+
+	serviceDb, err := database.GetServiceDbByName(ctx, helper, instance.Name, instance.Namespace)
```

Comment:
@stuggi not completely sure about the keystoneApi instance.name and Namespace usage .
This _should_ give me the service name and namespace for the getServiceDbByName so it can get the pointer to the DB object itself from it . is will that work ?

Reply:
@stuggi ^^ ?